crates/redstone/src/helpers/trie.rs

```diff
@@ -0,0 +1,174 @@
+use crate::network::error::Error;
+use crate::FeedId;
+
+const CHAR_START: u8 = 48; // Start at ASCII b'0'
+const CHAR_END: u8 = 90; // End at ASCII b'Z'
```

Comment:
Are we sure we dont use other characters for feeds?

Reply:
Good question. Lets discuss it. I am not sure if there is any Feed ID that will require special character. Nevertheless we can account for all the characters in basic ASCII. (Is there any Feed using UTF8 encoding?)
Before changing the Trie algorithm we can agree first if we want to keep it or not. If https://github.com/rust-lang/hashbrown is something that we want to pull in to the package then adjusting Trie makes no sens.